qtop.py

```diff
@@ -435,23 +449,38 @@ def print_wnid_lines(start, stop, highest_wn, hxxxx):
             print line_with_separators(d[line][start:stop], SEPARATOR, options.WN_COLON) + end_label.next()
 
     elif NAMED_WNS or options.FORCE_NAMES:  # names (e.g. fruits) instead of numbered WNs
-        raise NotImplementedError
-        just_name_dict = {}
-        color = 0
-        highlight = {0: 'cmsplt', 1: 'Red'}  # should obviously be customizable
-
-        for line, _ in enumerate(highest_wn):
-            just_name_dict[line] = ''
-        for column, _1 in enumerate(node_dict['wn_list']):
-            for line, _2 in enumerate(highest_wn):
-                try:
-                    letter = node_dict['wn_list'][column][line]
-                except TypeError:
-                    letter = ' '
-                just_name_dict[line] += colorize(letter, highlight[color])
-            color = 0 if color == 1 else 1
-        for line, _ in enumerate(highest_wn):
-            print just_name_dict[line] + '={__WNID__}'
+        # hosts = [state_corejob_dn['host'] for _, state_corejob_dn in node_dict['wn_dict'].items()]
+        # this is recalculated in calc_all_wnid_label_lines, need to refactor  # TODO
+        # node_str_width = len(max(hosts, key=len))
+
+        appends = {
+            '1': ['={__WNID__}'],
+            '2': ['={_Worker_}', '={__Node__}'],
+            '3': ['={_Worker_}', '={__Node__}', '={___ID___}'],
+            '4': ['={________}', '={_Worker_}', '={__Node__}', '={___ID___}'],
+            '5': ['={__FOR___}', '={__SALE__}', '={_Worker_}', '={__Node__}', '={___ID___}'],
+            '6': ['={_SPACE__}', '={__FOR___}', '={__SALE__}', '={_Worker_}', '={__Node__}', '={___ID___}'],
+            '7': ['={__PLACE_}', '={__YOUR__}', '={_ADVERT_}', '={__HERE__}', '={_Worker_}', '={__Node__}', '={___ID___}'],
```

Comment:
shouldn't this be configurable in a file, rather have hardwired strings as part of the code?